src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random randomNumberService = new Random();
+    private final ColorSupplier colorSupplierService = new ColorSupplier();
```

Comment:
I'd say that now your field names are a bit too verbose, it is fine if we name it just as they are and do not add anything else, I think we can call it a good practice to name classes like this just by their name, otherwise it may cause confusion
```suggestion
    private Random random = new Random();
    private final ColorSupplier colorSupplier = new ColorSupplier();
```